### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/lore`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4571]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4571|
|Type|enhancement|
|Created|2010-07-11 07:01:51Z|
|Branch|https://github.com/twisted/twisted/tree/sphinx-lore-4571|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for `projects/lore`

Attachments:

* [lore.xhtml.path](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/071/0716f42cf2489214bbefa4ae25166e41b9598b41/127947bd5a21d7350427734961c5e8695a96a998.path) (9766 bytes) - added by jdb on 2010-09-09 15:43:10Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4571 4571
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_sphinx_lore_4571 branches/sphinx-lore-4571
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1278831711000000 1278831711000000
changetime__1284816834000000 1284816834000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jdb's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|jdb commented|
|-|-|

Modified the xhtml input (made sure no <code> or <em> tag begins a newline), to correct the rst output. Now, there is no more missing space between a word and either or ** in the html final output.

**Comment by twisted-trac** [comment]
|<img alt="jdb's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|jdb removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [30027]) Branching to 'sphinx-lore-4571'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [30028]) Apply lore.xhtml.path 

refs [#4571](https://github.com/twisted/twisted/issues/4571)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks. This looks good. It highlights a lore2sphinx issue, I suppose - those comments at the end of the extending lore document should really be dropped or preserved as comments, not dumped into the output. But that should be fixed in lore2sphinx.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [30030]) Merge sphinx-lore-4571

Author: jdb
Reviewer: exarkun
Fixes: [#4571](https://github.com/twisted/twisted/issues/4571)

Tweaks to the Lore markup in the Lore documentation to make it
more amenable to conversion to Sphinx.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
